_source/_posts/2020-02-07-transactional-mfa.md

```diff
@@ -40,31 +40,31 @@ Did you know that outside of the standard authentication scenarios, a developer
 >3. A free [Okta Developer](https://developer.okta.com/) account
 >4. An existing [NodeJS Express Okta project](https://github.com/okta/samples-nodejs-express-4/tree/master/okta-hosted-login) where you will inject your on-demand MFA.
 
-## Create a Custom SAML Application in Okta
+## Enable a Factor in Okta
 
-In this example, you will be using a new early-access feature called [Step-up authentication with Okta Session](https://developer.okta.com/docs/reference/api/authn/#step-up-authentication-with-okta-session). This is a new feature that allows you to get a `stateToken` from Okta and use it as a parameter within the Okta Sign-In widget so that you can bootstrap the Okta Sign-In widget to do MFA automatically. If you don't have this feature enabled, I would suggest creating a support ticket to have this feature enabled on your Okta tenant.
+>Caution: If you are using the Developer Console, you need to first switch to the Admin Console (Classic UI). If you see Developer Console in the top left of the page, click it and select Classic UI to switch.
 
-To do this, you will need to create a custom SAML application.
+Before you configure an app in Okta to use multifactor, you must have at least one factor enabled. To do this, browse to: **Security > Multifactor**.
 
-Caution: If you are using the Developer Console, you need to first switch to the Admin Console (Classic UI). If you see Developer Console in the top left of the page, click it and select Classic UI to switch.
+Select **Okta Verify** and switch the dropdown from **Inactive** to **Active**. Click **Edit** and check the `Enable Push Notification` and `Require TouchId for Okta Verify` checkboxes. Click **Save**.
 
-Developer Console:
+{% img blog/transactional-mfa/okta-verify.png alt:"Okta Verify" width:"800" %}{: .center-image }
 
-{% img blog/transactional-mfa/developer-console.png alt:"Developer Console" width:"800" %}{: .center-image }
+## Create a Custom SAML Application in Okta
 
-Classic UI Console:
+In this example, you will be using a new early-access feature called [Step-up authentication with Okta Session](https://developer.okta.com/docs/reference/api/authn/#step-up-authentication-with-okta-session). This is a new feature that allows you to get a `stateToken` from Okta and use it as a parameter within the Okta Sign-In widget so that you can bootstrap the Okta Sign-In widget to do MFA automatically. If you don't have this feature enabled, I would suggest creating a support ticket to have this feature enabled on your Okta tenant.
 
-{% img blog/transactional-mfa/classic-console.png alt:"Classic Console" width:"800" %}{: .center-image }
+To do this, you will need to create a custom SAML application.
 
-Click **Applications** -> **Add Application**. Next, click **Add Application** -> **Create New App** -> **Select Web and SAML**.
+Click **Applications** > **Applications**. Next, click **Add Application** > **Create New App** > **SAML 2.0**. Click **Create**.
```

Comment:
This seems wrong. Double Applications?

Reply:
yeah... It's how the developer ui is

Reply:
I meant the classic ui

Reply:
![image](https://user-images.githubusercontent.com/364562/74073080-d0443900-49bd-11ea-8432-43c3e134bcf9.png)